config/locales/en.yml

```diff
@@ -666,6 +666,7 @@ en:
       use_wiki: Enable wiki
       webstats_tracking_code: Tracking code
     tabs:
+      applications: Apps
```

Comment:
you can use `Apps` for german too. (long version would be `Anwendungen`, but might be not as clear as the short version)